src/TwoBucks/UpdateProfile.java

```diff
@@ -84,24 +105,24 @@ public static void readTxtFile(String[] args) throws Exception{
             }
 
             if (flag == true){
-                BufferedWriter buffWriter = new BufferedWriter(new FileWriter(txtFile));
+                BufferedWriter buffWriter = new BufferedWriter(new Filewriter(txtFile));
 
                 for (i=0; i<numOfRows; i++){
-                    buffWriter.write(String.join(",",dataRows[i]) + "\n");
+                    buffWriter.write(String.join(",",dataRows[]) + "\n");
                 }
                 buffWriter.close();
             }
             else{
-                System.out.println("Last name not found.");
+                System.out.println("Last name not found.")
             }
 
         }else if(option == 3){
             int rowIndex = 2;
 
             System.out.println("Please confirm your existing email: ");
-            String oldEmail = scnr.next();
+            String oldEmail = input.next();
             System.out.println("Please enter your new email: ");
-            String newEmail = scnr.next();
+            String newEmail = input.next();
```

Comment:
consider using setEmail method from the User class

Reply:
Done